src/Resources/Locales/en.json

```diff
@@ -583,6 +583,61 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "dataHistory": {
+            "title": "Data history widget",
+            "libraryName": "Data history",
+            "description": "Show timeseries data in context of your 3D scene",
+            "notEnabledDescription": "Data history is not enabled for this ADT instance.",
+            "form": {
+                "connectionStringTitle": "Connection string",
+                "connectionStringPlaceholder": "Enter connection string (e.g. key1=value1;key2=value2;key3=value3)",
+                "connectionStringInformation": "Data history widgets target the Azure Data Explorer instance connected to the Azure Digital Twins instance at the time of creation.",
+                "connectionLoadingText": "Loading connection information...",
+                "noConnectionInformationText": "No connection information found",
+                "displayNamePlaceholder": "Enter display name",
+                "timeSeries": {
+                    "timeSeriesDescription": "Add up to {{maxTimeSeriesCount}} timeseries to display",
+                    "title": "Time series",
+                    "description": "A series to display in the line chart",
+                    "property": "Property",
+                    "unit": "Unit",
+                    "label": "Label",
+                    "labelPlaceholder": "Enter label for this series to show in chart",
+                    "add": "Add time series",
+                    "edit": "Edit time series",
+                    "remove": "Remove time series"
+                },
+                "chartOptions": {
+                    "yAxisType": {
+                        "description": "Chart option to set if each series has separate (Independent) y-axes or sharing the same y-axis (Shared)",
+                        "label": "Y axis type",
+                        "sharedLabel": "Shared",
+                        "independentLabel": "Independent"
+                    },
+                    "quickTimeSpan": {
+                        "label": "Default time span",
+                        "options": {
+                            "Last 15 mins": "Last 15 mins",
```

Comment:
where do we use these loc versions? I thoguht I saw they were hardcoded in an enum

Reply:
Because we need to localize the quick span options. The quick span options itself is an enum type and this enum is used to generate a mapping for millisecond number value for each enum key.

Reply:
Doesn't look like they are getting localized.
![image](https://user-images.githubusercontent.com/57726991/195202033-43693ff7-8eb4-4850-b39b-ea37f88187df.png)

Reply:
Same in aggregation type.
![image](https://user-images.githubusercontent.com/57726991/195202099-377bd64f-d262-4ebc-9d91-5bfc5f27ce8a.png)

Reply:
off I blame i18next, fixed the translation: 
![image](https://user-images.githubusercontent.com/45217314/195222759-6a1f473b-ae05-44d2-8c1b-bb6c931c769e.png)

Reply:
by the way aggregation type is **NOT** going to be localized since it is a numeric operations on data which is universal. This is what we also did in TSI app.